examples/yolo/yolov3-tiny.cpp

```diff
@@ -114,6 +157,28 @@ static bool load_labels(const char * filename, std::vector<std::string> & labels
     return true;
 }
 
+static bool load_labels_gguf(const struct gguf_context * ctx, const char * filename, std::vector<std::string> & labels)
+{
+    int tensor = gguf_find_tensor(ctx, filename);
+    if (tensor == -1) {
+        return false;
+    }
+    const size_t offset = gguf_get_tensor_offset(ctx, tensor);
+    const size_t len = gguf_get_tensor_size(ctx, tensor);
```

Comment:
Somehow I didn't notice this before: `gguf_get_tensor_size()` is not needed too. You can instead use:

```suggestion
    const size_t len = ggml_nelemens(tensor);
```

So remove `gguf_get_tensor_size` all together

Reply:
okay, i removed gguf_get_tensor_size from ggml.h, ggml.c, yolov3-tiny.cpp.